app/scripts/models/account.js

```diff
@@ -96,7 +96,8 @@ define([
             self.set('verified', verified);
           }, function () {
             // Ignore errors; we'll just fetch again when needed
-          });
+            // TODO Remove once https://github.com/eslint/eslint/issues/3279 gets fixed
```

Comment:
@vladikoff The issue is getting fixed soon, i see no other reason to not update.